### PR TITLE
Ar 36/blog add new article

### PR DESCRIPTION
**Pull request by vlad-tymoshchyk-imprivata**
Implemented creating of new and proper editing of existing blog articles

### CODE REVIEW

File: Frontend/src/containers/admin/Home/store/saga/index.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ import { fetchLocations } from '../../../../../api/admin/locations';
 
 function* fetchHomePageAnimalsListSaga(action: { type: string, payload?: IRequestParams }) {
     try {
-        const response = yield call(fetchAdminAnimals,action.payload);
-        yield put(actionAdminHomeFetchAnimalsSuccess(response))
+        const filter = yield call(fetchAdminAnimals,action.payload);
```
Comment (zherng) [inline Frontend/src/containers/admin/Home/store/saga/index.ts]: Is this change really needed? I am afraid that another functionality will be broken.

Comment (vlad-tymoshchyk-imprivata) [reply Frontend/src/containers/admin/Home/store/saga/index.ts]: No, it's not needed, missed that when reviewed my changes.
Fixed it.
Thanks for noticing.

File: Frontend/src/containers/admin/AnimalEditCard/ui/AnimalEditCard.tsx [hunk 2]
```diff
@@ -191,9 +191,19 @@ class AnimalEditCard extends React.Component<IPropTypes> {
...
     })
+    // @ts-ignore
+    window.coverimage = coverImage;
+    // @ts-ignore
+    window.dothis = () => {
+      this.setState({
+        coverImage
+      })
+    }
+    console.log('coverImage change after', coverImage);
```
Comment (zherng) [inline Frontend/src/containers/admin/AnimalEditCard/ui/AnimalEditCard.tsx:206]: Please remove the extra "console.log" that was used for debugging
